lib/inc/facter/facts/fact.hpp

```diff
@@ -684,6 +684,11 @@ namespace facter { namespace facts {
          * The fips mode fact for RHEL/Linux* facts.
          */
         constexpr static char const* fips_enabled = "fips_enabled";
+
+        /**
+         * The fact for Aix NIM type (master/client)
```

Comment:
master/standalone instead of client?

Reply:
Not really. The NIM package manager structure is composed from one master and multiple clients. Standalone is just a keyword from the config file.